inst/extdata/OSD/A/ALSCO.json

```diff
@@ -125,6 +125,42 @@
         "topography": "wavy",
         "narrative": "Btk1--5 to 11 inches; brown (7.5YR 4/4) very cobbly clay loam, dark brown (7.5YR 3/4) moist; moderate fine and medium subangular blocky structure; slightly hard, friable, moderately sticky and moderately plastic; many very fine and fine roots; many very fine and fine interstitial and common very fine and fine tubular pores; common thin clay films on faces of peds and in pores; 30 percent cobbles and 25 percent gravel; few fine lime segregations on underside of rock fragments; strongly effervescent (5 percent calcium carbonate); moderately alkaline (pH 8.2); clear wavy boundary (6 to l4 inches thick)."
       },
+      {
```

Comment:
In some cases entire horizons were missing, now top or bottom depth may be NA but other data from that layer is returned. These missing depths are source of some "new" errors in RO error reporting